app/uk.gov.hmrc.helptosaveproxy/connectors/DWPConnector.scala

```diff
@@ -68,18 +70,25 @@ class DWPConnectorImpl @Inject()(
 
     val timeContext: Timer.Context = metrics.dwpClaimantCheckTimer.time()
 
+//    val queryParams = Seq(
+//      "systemId"        -> appConfig.systemId,
+//      "thresholdAmount" -> threshold.toString)
+
     val queryParams = Seq(
       "systemId"        -> appConfig.systemId,
-      "thresholdAmount" -> threshold.toString)
+      "thresholdAmount" -> threshold.toString,
+      "transactionId"   -> transactionId.toString)
 
-//    val queryParams = Seq(
-//      "systemId"        -> appConfig.systemId,
-//      "thresholdAmount" -> threshold.toString,
-//      "transactionId"   -> transactionId.toString)
+    val isNewDwpEndpoint = FEATURE("isNewDwpEndpoint", appConfig.runModeConfiguration, logger).enabled
+
+    val dwpUrl = isNewDwpEndpoint match {
+    case true => s"${appConfig.newDwpUrl}/$nino"
+    case _ => s"${appConfig.newDwpUrl}/$nino"
```

Comment:
⚠️ These two lines seem identical. You sure there isn't an error? If not, please combine.
⛏️  Indentataion, format document.